packages/core/src/browser/style/hover-service.css

```diff
@@ -29,7 +29,12 @@
   border: 1px solid var(--theia-editorHoverWidget-border);
   padding: var(--theia-ui-padding);
   max-width: var(--theia-hover-max-width);
-  z-index: 1000;
```

Comment:
Since not all browser fully support the new API, does it make sense to still keep this or what is the argument for removing it. (Though if we keep it, extracting it into it's own variable similar to `--theia-hover-max-width` would be nice)

Reply:
I didn't encounter any issues myself but officially, the `hint` value that we are using does not seem to be fully supported by all browsers:
https://developer.mozilla.org/en-US/docs/Web/API/Popover_API#api.htmlelement.popover. So I thought keeping that as a fallback might be useful.

Reply:
Value "hint" is kind of forward proofing. Unknown value is treated as auto AFAIK.
And in Theia code we still manage hiding previous hovers ourselves.
So technically there is no difference. They all still go to top layer and z-index is not needed.

Reply:
Since I also didn't encounter any issues, I'm fine with removing it. Thank you!